var/spack/repos/builtin/packages/py-py2cairo/package.py

```diff
@@ -33,9 +33,13 @@ class PyPy2cairo(Package):
 
     version('1.10.0', '20337132c4ab06c1146ad384d55372c5')
 
+    variant('X', default=True,
+            description='Enable cairo X11 dependency')
+
```

Comment:
why should the variant be removed? Isn't that the usecase? One variant uses the X11 backend the other doesn't?